src/SecureHeadersMiddleware.cs

```diff
@@ -64,6 +64,12 @@ public async Task Invoke(HttpContext httpContext)
                     _config.ContentSecurityPolicyConfiguration.BuildHeaderValue());
             }
 
+            if (_config.UseXContentSecurityPolicy)
+            {
+                httpContext.TryAddHeader(Constants.XContentSecurityPolicyHeaderName,
```

Comment:
if this is write if the tryaddHeader fails we get nothing? (maybe a log warning or error would be nice)

Reply:
I don't think we should change the default behavior of the package in this PR. If we want to fix this we should do this in another PR. But be aware that this middleware does not use any logger and I think we should keep it as tight as possible as it is used in every request.

Reply:
okay